.builders/images/macos-arm64/extra_build.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -exu
+
+# Packages which must be built from source
+always_build=()
+
+if false && [[ "${DD_BUILD_PYTHON_VERSION}" == "3" ]]; then
```

Comment:
Disabled the rebuild of `confluent-kafka` and `librdkafka` because some module doesn't support universal2 wheels and I didn't managed to force take the arm64 wheel instead of universal

Reply:
I don't think it is necessary for macOS m1 to have `kerberos` support at least for now since the main goal is to allow easier local building on our laptops